dmsite/venv/src/Authentication/NewAccountTextField.js

```diff
@@ -0,0 +1,186 @@
+        );
+
+        const websiteOptions = autoCompleteResult.map(website => (
+            <AutoCompleteOption key={website}>{website}</AutoCompleteOption>
+        ));
+
+        return (
+            <Form onSubmit={this.handleSubmit}>
+                <Form.Item
+                    {...formItemLayout}
+                    label="E-mail"
+                >
+                    {getFieldDecorator('email', {
+                        rules: [{
+                            type: 'email', message: 'The input is not valid E-mail!',
+                        }, {
+                            required: true, message: 'Please input your E-mail!',
+                        }],
+                    })(
+                        <Input />
+                    )}
+                </Form.Item>
+                <Form.Item
+                    {...formItemLayout}
+                    label="Password"
+                >
+                    {getFieldDecorator('password', {
+                        rules: [{
+                            required: true, message: 'Please input your password!',
+                        }, {
+                            validator: this.validateToNextPassword,
+                        }],
+                    })(
+                        <Input type="password" />
+                    )}
+                </Form.Item>
+                <Form.Item
+                    {...formItemLayout}
+                    label="Confirm Password"
+                >
+                    {getFieldDecorator('confirm', {
+                        rules: [{
+                            required: true, message: 'Please confirm your password!',
+                        }, {
+                            validator: this.compareToFirstPassword,
+                        }],
+                    })(
+                        <Input type="password" onBlur={this.handleConfirmBlur} />
+                    )}
+                </Form.Item>
+                <Form.Item
+                    {...formItemLayout}
+                    label="Captcha"
+                    extra="We must make sure that your are a human."
+                >
+                    <Row gutter={8}>
+                        <Col span={12}>
+                            {getFieldDecorator('captcha', {
+                                rules: [{ required: true, message: 'Please input the captcha you got!' }],
+                            })(
+                                <Input />
+                            )}
+                        </Col>
+                        <Col span={12}>
+                            <Button>Get captcha</Button>
+                        </Col>
+                    </Row>
+                </Form.Item>
+                <Form.Item {...tailFormItemLayout}>
+                    {getFieldDecorator('agreement', {
+                        valuePropName: 'checked',
+                    })(
+                        <Checkbox><a href=""> I have read the agreement</a></Checkbox>
```

Comment:
I think the captcha and the ToS agreement are a little unnecessary, since this is just a class project (and we really don't have a ToS anyway lol), so I don't know if these sections are really necessary. We can leave them in, but I don't think we need to really develop them any further.

Reply:
I commented those out just in case we want to add them in the future!